test/quantization/core/test_quantized_op.py

```diff
@@ -4035,6 +4035,74 @@ def test_qlinear_tanh(self):
                                         use_bias, post_op, use_multi_dim_input,
                                         use_channelwise)
 
+    @skipIfNoONEDNN
+    def test_qlinear_with_cpu_tensor_onednn(self):
+        '''
+        Prepack
+        - input: weight is an int8 tensor from CPU backend
+        - output: packed weight and bias as MKLDNN tensors
+
+        Linear
+        - inputs: packed weight and bias as MKLDNN tensors
+        - output: linear result as an int8 tensor from CPU backend
+        These ops are used for Inductor quantization
+        '''
+        qlinear_prepack = torch.ops.quantized.linear_prepack_mkldnn
+        qlinear = torch.ops.quantized.linear_with_mkldnn_weight_bias
```

Comment:
can we align the names, e.g. rename mkldnn to onednn?

Reply:
Thanks for the suggestion. I will have it fixed after the onednn update and rebase.